misc.txt

```diff
@@ -0,0 +1,83 @@
+List(
+    Label("start"){
+        List(
+            Face(WEST),
+            Go until FRONT_WALL,
+            Face(NORTH),
+            Go until FRONT_WALL,
```

Comment:
I like this idea to go until front wall, but when implementing the right-hand rule, how do you specify something complex like hugging the wall on your right? I imagine it would be something like "Go until [condition]", but I'm not too sure how writing the condition would work out.